packages/components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -77,8 +81,10 @@ export const Tooltip = ({
     disabled,
     className,
     isFullWidth = false,
+    isInline = false,
     isOpen,
     hasArrow,
+    hasIcon = false,
```

Comment:
:thought_balloon:  The `TooltipSymbol`, now deprecated, supports specific `icon` other than `question`. Although I can see that prop is not used anywhere at the moment, so there is no point in supporting it.
But here, if the icon is always `question`, the prop could be named `hasQuestionIcon` to make it clear.